build.sh

```diff
@@ -25,10 +25,10 @@ cat "$_root_dir/flags.portable.gn" >> "$_src_dir/out/Default/args.gn"
 
 # Set commands or paths to LLVM-provided tools outside the script via 'export ...'
 # or before these lines
-export AR=${AR:=llvm-ar}
-export NM=${NM:=llvm-nm}
-export CC=${CC:=clang}
-export CXX=${CXX:=clang++}
+export AR=gcc-ar
+export NM=gcc-nm
+export CC=gcc
+export CXX=g++
```

Comment:
We should not hardcode these in case the user needs to change the compiler via environment variables.

Reply:
They were hardcoded since the gcc patchset was added to allow building on a pure gcc toolchain without llvm/clang. I have dropped that patchset now and only kept the real fixes.
The CI is still failing though, because of a musl patch. I'll look into it in some time